apps/fhir/bluebutton/tests/test_wellknown_endpoints.py

```diff
@@ -0,0 +1,149 @@
+from apps.test import BaseApiTest
+
+from collections import namedtuple as NT
+from django.conf import settings
+from django.test.client import Client
+from httmock import all_requests, HTTMock
+from oauth2_provider.models import get_access_token_model
+import requests
+from unittest import skipIf
+
+# Introduced in bb2-4184
+# Rudimentary tests to make sure endpoints exist and are returning
+# basic responses that make sense. Not a deep test.
+#
+# PRECONDITION
+# You must be on the VPN. Tested against TEST w.r.t. certs.
+
+
+Appropriate = NT('Status', 'url,version,status_code')
+Status = NT('Status', 'url,status_code')
+ContainsVersion = NT('ContainsVersion', 'url,version,path')
+MissingFields = NT('MissingFields', 'url,version,fields,status_code')
```

Comment:
I'm not sure about this name, maybe it's just me, but 'missing' implies incorrect to me. Might be solved by a way-too-wordy rename like `ConfirmFieldsNotPresentInResponse`, or might be more easily addressed by adding a one-line comment describing the intention of each of these test types.